css-images-4/Overview.bs

```diff
@@ -2393,6 +2465,39 @@ Interpolating <<gradient>> {#interpolating-gradients}
 		Then, for each pair of color-stops,
 		interpolate the position and color independently.
 
+Interpolating <<1d-image>> {#interpolating-1d-images}
+-----------------------------------------------
+
+	Similar to <a lt="interpolating-gradients">gradient images</a>,
+	one-dimensional images can be interpolated directly in CSS transitions and animations,
+	smoothly animating from one image to another.
+	There are only a few restrictions on what images are allowed to be interpolated:
+
+	1. Both the starting and ending image must have the same number of <<color-stripe>>s.
+
+	2. Neither image uses a combination of <<length>>, <<percentage>>, and <<flex>> stripes.
+
+	If the two image satisfy both constraints,
+	they must be interpolated as described below.
+	If they fail the second one only,
+	they must be abruptly transitioned at 50%
+	(unless otherwise specified by a future specification).
+	If they fail the first constraint,
+	they must be interpolated using ''cross-fade()''
+	as for generic images.
+
+	Note: The abrupt transition at 50% is so that content will not rely on cross-fading,
+	and smarter interpolation rules can be added for this case in the future.
+
+	1. Interpolate each component and stripe of the images independently.
+
+	2. To interpolate a stripe,
+		first match each stripe in the start image
+		to the corresponding stripe at the same index in the end image.
+		Then, for each pair of stripes,
+		interpolate the <a href="https://www.w3.org/TR/css3-transitions/#animtype-length">thickness</a>
```

Comment:
How to interpolate a `<length-percentage>` with a `<flex>`?

Rather than checking that each image doesn't mix different types, should the requirement be that each pair of matched stripes have the same type?

Reply:
https://github.com/w3c/csswg-drafts/issues/8163